compiler/rustc_const_eval/src/interpret/call.rs

```diff
@@ -730,6 +730,7 @@ impl<'tcx, M: Machine<'tcx>> InterpCx<'tcx, M> {
         let existential_trait_ref = ty::ExistentialTraitRef::erase_self_ty(tcx, virtual_trait_ref);
         let concrete_trait_ref = existential_trait_ref.with_self_ty(tcx, dyn_ty);
 
+        let _span = enter_trace_span!(M, resolve::expect_resolve_for_vtable, ?def_id);
```

Comment:
Did we call this `_span` everywhere? I now realize that's confusing because usually that refers to the `Span` type...

Reply:
Maybe `_s` would be better? The name is not relevant anyway, we just need it so that the returned value is not immediately destructed